nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -420,18 +424,24 @@ def _impl(inputs, attr, params):
             ignores=['index_type', 'T'])(new_inputs, attr)
     return _impl
 
+def _split():
+    def _impl(inputs, attr, params):
+        pop_node = inputs.pop(0)
+        axis = params[pop_node.list_output_names()[0]].asnumpy()[0]
+        return AttrCvt(op_name="split",
+                       ignores=['num_split'],
+                       extras={'indices_or_sections':attr['num_split'],
+                               'axis': axis})(inputs, attr)
+    return _impl
+
 def _lrn():
     def _impl(inputs, attr, params):
-        new_inputs = []
-        attr_new = {}
-        depth_radius = attr.get('depth_radius', 5)
-        size = (depth_radius * 2) + 1
-        attr_new['axis'] = 3 # Fix axis, NHWC format
-        attr_new['size'] = size
-        attr_new['bias'] = attr.get('bias', 1)
-        attr_new['alpha'] = attr.get('alpha', 1) * size
-        attr_new['beta'] = attr.get('beta', 0.5)
-        return AttrCvt(op_name='lrn')(new_inputs, attr_new)
+        depth_radius = attr.get('depth_radius', 2)
```

Comment:
LRN is already implemented earlier. Any reason for this modify ?

Reply:
Previously implemented LRN was wrong as per TF formula and input tensor is never pass to operator, this leak observe after run the [AlexNet](https://www.cs.toronto.edu/~guerzhoy/tf_alexnet/) model

Reply:
I think that was a bug passing empty inputs ```new_inputs = []```
Removing new_inputs and using inputs instead should work for alexnet as well.
Keep the remaining implementation default value handling.

@siju-samuel  can you confirm the depth_radius default value as 5 which results size=11 ? or is it a typo !!

```
   depth_radius = attr.get('depth_radius', 5)
   size = (depth_radius * 2) + 1
``` 

Test case was passed as the input was taken from params instead.
#1546 should fix this issue by replacing all const inputs to Placeholders.

Reply:
@srkreddy1238  as per tensorflow, the default value of depth_radius is 5. So the size will become 11.
https://www.tensorflow.org/api_docs/python/tf/nn/local_response_normalization